### PR TITLE
Mention support for Bot API 7.6

### PR DESCRIPTION
**Pull request by KnorpelSenf**
https://core.telegram.org/bots/api#july-1-2024

### CODE REVIEW

**Review by AndriiKuhaivskyiRadency — APPROVED** [review]
:shipit:

**Review by LWJerri — APPROVED** [review]
:shipit:

File: site/docs/uk/README.md [hunk 1]
```diff
@@ -99,8 +99,8 @@ bot.start();
 
 <div style="font-size: 0.75rem; display: flex; justify-content: center;">
 
-© 2021-2024 &middot; grammY підтримує Telegram Bot API 7.5, який був [випущений](https://core.telegram.org/bots/api#june-18-2024) 18-го червня 2024 року.
-Остання зміна: редагування бізнес-повідомлень і перегляд Star транзакцій.
+© 2021-2024 &middot; grammY підтримує Telegram Bot API 7.6, який був [випущений](https://core.telegram.org/bots/api#july-1-2024) 1-го липня 2024 року.
+(Остання зміна: Платні медіа)
```
Comment (niusia-ua) [inline site/docs/uk/README.md:103]:
```suggestion
Остання зміна: платні медіа
```

**Review by niusia-ua — CHANGES_REQUESTED** [review]

**Review by niusia-ua — APPROVED** [review]

**Review by roziscoding — APPROVED** [review]

**Comment by LWJerri** [comment]
123

**Comment by AndriiKuhaivskyiRadency** [comment]
123

**Comment by KnorpelSenf** [comment]
I am fairly confident that these changes are correct. If Spanish or Indonesian are off, please submit a fix in a subsequent PQ. I will merge this.

**Review by quadratz — APPROVED** [review]

**Pull request merged**